package.json

```diff
@@ -16,7 +16,7 @@
     "build": "rollup -c",
     "start": "rollup -c -w",
     "prepare": "npm run build",
-    "react-start": "react-scripts start",
```

Comment:
`npx react-scripts start` didn't seem to work for me. Let me know if you're still using it.